Resources/Textures/_DV/Clothing/OuterClothing/Suits/engineering-light.rsi/meta.json

```
@@ -0,0 +1,54 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Edited by EmberAstra. Based on the base Engineering Hardsuit sprites.",
```

This is such a nitpik but I think better safe than sorry where copyright is concerned.

Let's copy over the full copyright string. In the extremely unlikely event that the engineering hardsuit gets removed or re-sprited from scratch, chasing down the original work this was derived from could be tough. With copyright you generally want all the info readily available.

```suggestion
    "copyright": "Edited by EmberAstra. Based on the Hardsuits/engineering.rsi sprites which have the following copyright: Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e. Vox state made by Flareguy for SS14 | harpy version by VMSolidus, equipped-OUTERCLOTHING-dog modified from equipped-OUTERCLOTHING by Sparlight (GitHub).",
```

Or something like that.